### PR TITLE
[BACK-1278] Remove queries/indexes on `_schemaVersion` from `tide-whisperer`

### PR DESCRIPTION
**Pull request by pazaan**

### CODE REVIEW

File: store/store.go [hunk 1]
```diff
@@ -360,15 +360,11 @@ func (c *MongoStoreClient) HasMedtronicDirectData(userID string) (bool, error) {
 
 	opts := options.FindOne()
```
Comment (toddkazakov) [inline store/store.go:361]: Do you need those empty options?

File: store/store.go [hunk 2]
```diff
@@ -427,15 +423,11 @@ func (c *MongoStoreClient) HasMedtronicLoopDataAfter(userID string, date string)
...
```
Comment (toddkazakov) [inline store/store.go:426]: Do you need those empty options?

File: store/store.go [hunk 3]
```diff
@@ -392,21 +391,17 @@ func (c *MongoStoreClient) GetDexcomDataSource(userID string) (bson.M, error) {
 		},
 	}
 
-	dataSources := []bson.M{}
-	opts := options.Find().SetLimit(1)
-	cursor, err := c.client.Database("tidepool").Collection("data_sources").Find(c.context, query, opts)
+	dataSources := bson.M{}
+	err := c.client.Database("tidepool").Collection("data_sources").FindOne(c.context, query).Decode(&dataSources)
```
Comment (toddkazakov) [inline store/store.go:397]:
```suggestion
	err := c.client.Database("tidepool").Collection("data_sources").FindOne(c.context, query).Decode(&dataSource)
```

**Comment by pazaan** [comment]
@toddkazakov - sorry. Requires a re-review. Because of the messy branch history, and https://github.com/tidepool-org/tide-whisperer/pull/86 making it into `master` before this made it to `master`, it needed a merge instead of a simple ff-merge.

**Comment by derrickburns** [comment]
@pazaan  could you rebase and forced push too avoid a merge?

**Pull request merged**